src/styles/generic/fonts.story.js

```diff
@@ -46,6 +46,62 @@ stories.add('fonts', () => ({
         <p style={style}>
           ABCČĆDĐEFGHIJKLMNOPQRSŠTUVWXYZŽabcčćdđefghijklmnopqrsštuvwxyzžАБВГҐДЂЕЁЄЖЗЅИІЇЙЈКЛЉМНЊОПРСТЋУЎФХЦЧЏШЩЪЫЬЭЮЯабвгґдђеёєжзѕиіїйјклљмнњопрстћуўфхцчџшщъыьэюяΑΒΓΔΕΖΗΘΙΚΛΜΝΞΟΠΡΣΤΥΦΧΨΩαβγδεζηθικλμνξοπρστυφχψωάΆέΈέΉίϊΐΊόΌύΰϋΎΫΏĂÂÊÔƠƯăâêôơư1234567890‘?’“!”(%)[#]@/&\-+÷×=®©$€£¥¢:;,.*
         </p>
+        <div class="tw-mini-header">English</div>
```

Comment:
You can remove the lorem ipsum paragraphs now, and also the paragraph with special characters. Most of them are already covered in specific languages.

I'd create at the bottom and section "Special characters" and only add remaining that are not covered:
`1234567890‘?’“!”(%)[#]@/&\-+÷×=®©$€£¥¢:;,.*`